rewriter/Minitest.cc

```diff
@@ -485,8 +485,21 @@ ast::ExpressionPtr runSingle(core::MutableContext ctx, bool isClass, ast::Send *
 
     auto *block = send->block();
 
+    // Allow both self.describe and RSpec.describe calls (and other RSpec methods)
     if (!send->recv.isSelfReference()) {
-        return nullptr;
+        // Only allow RSpec calls for describe-like methods
+        if (send->fun == core::Names::describe() || send->fun == core::Names::context() ||
+            send->fun == core::Names::xcontext() || send->fun == core::Names::xdescribe()) {
+            if (auto constant = ast::cast_tree<ast::UnresolvedConstantLit>(send->recv)) {
+                if (constant->cnst != core::Names::Constants::RSpec()) {
```

Comment:
I'm not sure if this makes sense right now in practice, because once this merges, our repo won't be able to upgrade sorbet until rspec typing is up to snuff (or, we downgrade typed level for a ton of files and upgrade it as we add more typing).

I'm not sure what a better alternative is though – open to ideas. Maybe a feature flag in `sorbet/config`?